server/fleet/secrets.go

```diff
@@ -1,3 +1,24 @@
 package fleet
 
+import (
+	"fmt"
+	"strings"
+)
+
 const FLEET_SECRET_PREFIX = "FLEET_SECRET_"
```

Comment:
I don't think we use format for constant names. I recommend changing it to `ServerSecretPrefix` or similar.

Reply:
I can do that